src/overcooked_ai_py/visualization/pygame_utils.py

```diff
@@ -0,0 +1,86 @@
+import pygame
+from pygame.locals import HWSURFACE, DOUBLEBUF, RESIZABLE, QUIT, VIDEORESIZE
+from overcooked_ai_py.utils import load_from_json
+
+def run_static_resizeable_window(surface, fps=30):
+    """
+    window that can be resized and closed using gui
+    """
+    pygame.init()
+    clock = pygame.time.Clock()
+    window = pygame.display.set_mode(surface.get_size(), HWSURFACE|DOUBLEBUF|RESIZABLE)
+    window.blit(surface, (0, 0))
+    pygame.display.flip()
+    try:
+        while True:
+            pygame.event.pump()
+            event = pygame.event.wait()
+            if event.type == QUIT: 
+                pygame.display.quit()
+                pygame.quit()
+            elif event.type == VIDEORESIZE:
+                window = pygame.display.set_mode(event.dict['size'], HWSURFACE|DOUBLEBUF|RESIZABLE)
+                window.blit(pygame.transform.scale(surface, event.dict['size']), (0, 0))
+                pygame.display.flip()
+                clock.tick(fps)
+    except:
+        pygame.display.quit()
+        pygame.quit()
+        if event.type != QUIT:
+            raise
+
+def vstack_surfaces(surfaces, background_color=None):
```

Comment:
If possible maybe add a bit more documentation to this file? Just even a one line docstring for all functions would be good. I feel like these kinds of manipulations are usually a bit opaque otherwise

Reply:
Thanks for the feedback! I will add some docstrings today or tomorrow at the latest.

Reply:
Docstrings added